app/models/manageiq/providers/ibm_power_vc/cloud_manager.rb

```diff
@@ -370,6 +370,14 @@ def self.params_for_create
                     :label     => _("PowerVC Server Password"),
                     :type      => "password",
                   },
+
+                  {
+                    :component => "password-field",
```

Comment:
Is this "secret"? Does it need to be a password-field or just a simple text-field

Reply:
A simple field would suffice.

Reply:
done